src/tools/rest-backend/.gitignore

```diff
@@ -0,0 +1,6 @@
+#ignore unwanted cmake files
+CMakeCache.txt
+CMakeFiles/
+
+#ignore netbeans project files
+nbproject/
```

Comment:
Weird, in my code version it definitely has none (maybe it was added recently, there were some changes if I remember correctly). I can add nbproject/ also there.